drivers/led/lp5562.c

```diff
@@ -895,6 +904,66 @@ static inline int lp5562_led_off(const struct device *dev, uint32_t led)
 	return lp5562_led_set_brightness(dev, led, dev_data->min_brightness);
 }
 
+static int lp5562_verify_config(const struct lp5562_config *config)
+{
+	if (!IN_RANGE(config->r_current, LP5562_MIN_CURRENT_SETTING, LP5562_MAX_CURRENT_SETTING)) {
+		LOG_ERR("Set output current of red channel out of range.");
+		return -EINVAL;
+	}
+
+	if (!IN_RANGE(config->g_current, LP5562_MIN_CURRENT_SETTING, LP5562_MAX_CURRENT_SETTING)) {
+		LOG_ERR("Set output current of green channel out of range.");
+		return -EINVAL;
+	}
+
+	if (!IN_RANGE(config->b_current, LP5562_MIN_CURRENT_SETTING, LP5562_MAX_CURRENT_SETTING)) {
+		LOG_ERR("Set output current of blue channel out of range.");
+		return -EINVAL;
+	}
+
+	if (!IN_RANGE(config->w_current, LP5562_MIN_CURRENT_SETTING, LP5562_MAX_CURRENT_SETTING)) {
+		LOG_ERR("Set output current of white channel out of range.");
+		return -EINVAL;
+	}
```

Comment:
might want to use strings in the form "Set output current of %s channel out of range." to save some flash